.circleci/config.base.yml

```diff
@@ -473,6 +476,8 @@ jobs:
         type: executor
         default: l_large
     executor: << parameters.os >>
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
```

Comment:
this migration tests was missing this, all other migration tests were already specifying this